.github/terraform/scripts/verify_mancenter.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+# set -o pipefail
```

Comment:
do we need it? or why it is commented out?

Reply:
It is not compulsory but I thought it would help to have it. However when I had it, remote-exec always failed on VMs. So I commented it out. I will delete it.